# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-text2skin/framespersec.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./common.h.orig	2005-01-27 17:31:12.000000000 +0000
+++ ./common.h	2009-05-03 16:40:09.000000000 +0000
@@ -10,6 +10,10 @@
 #include <vdr/osd.h>
 #include <vdr/config.h>
 
+#ifndef FRAMESPERSEC
+#define FRAMESPERSEC 25
+#endif
+
 #if defined(DEBUG) || defined(BENCH)
 #	ifdef DEBUG
 #		define Dprintf(x...) fprintf(stderr, x)
--- ./display.h.orig	2005-01-26 20:43:01.000000000 +0000
+++ ./display.h	2009-05-03 16:40:23.000000000 +0000
@@ -11,6 +11,10 @@
 #include <vdr/skins.h>
 #include <string>
 
+#ifndef FRAMESPERSEC
+#define FRAMESPERSEC 25
+#endif
+
 class cText2SkinData;
 class cText2SkinI18n;
 class cText2SkinLoader;
